docroot/profiles/custom/cgov_site/modules/custom/cgov_schema_org/cgov_schema_org.module

```diff
@@ -11,7 +11,8 @@
 function cgov_schema_org_preprocess_html(&$variables) {
   $entity = _cgov_schema_org_get_curr_entity();
   if ($entity && $entity->hasField('field_schema_org_data')) {
-    $schemaOrgType = $entity->field_schema_org_data->page_itemtype;
+    /** @var \Drupal\Core\Entity\ContentEntityInterface $entity */
+    $schemaOrgType = $entity->get('field_schema_org_data')->getValue()[0]['page_itemtype'];
```

Comment:
Can you try `$entity->get('field_schema_org_data')->first()?->get('page_itemtype')`

Reply:
Resolved.